application/dash_application/utility/plot_builder.py

```diff
@@ -402,7 +403,7 @@ def dropdown():
             {"label": "Project", "value": PINERY_COL.StudyTitle},
             {"label": "Run", "value": PINERY_COL.SequencerRunName},
             {"label": "Kit", "value": PINERY_COL.PrepKit},
-            {"label": "Tissue Prep", "value": PINERY_COL.TissuePreparation},
+            {"label": "Tissue Material", "value": PINERY_COL.TissuePreparation},
```

Comment:
To confirm, everything but the column name should change 'prep' -> 'material'?

Reply:
Yes, that's correct